nautobot/utilities/utils.py

```diff
@@ -933,7 +933,7 @@ def convert_querydict_to_factory_formset_acceptable_querydict(request_querydict,
 def is_single_choice_field(filterset_class, field_name):
     # Some filter parameters do not accept multiple values, e.g DateTime fields, boolean fields, q field etc.
     field = get_filterset_field(filterset_class, field_name)
-    return isinstance(field, (DateFilter, DateTimeFilter, TimeFilter, BooleanFilter)) or field_name == "q"
+    return isinstance(field, (DateFilter, DateTimeFilter, NumberFilter, TimeFilter, BooleanFilter)) or field_name == "q"
```

Comment:
Actually, this might break `MultiValueNumberFilter` since it's a subclass of `NumberFilter`. Maybe we should invert this to be simply `not isinstance(field, django_filters.MultipleChoiceFilter)`?

Reply:
Probably incorrect as currently implemented for `MultiValueDateFilter`, `MultiValueDateTimeFilter`, `MultiValueTimeFilter` in fact...